crates/core/src/masp.rs

```diff
@@ -513,13 +582,23 @@ impl TransferTarget {
             _ => None,
         }
     }
+
+    /// Get the contained MaybeIbcAddress, if any
```

Comment:
What is a `MaybeIbcAddress` ?

Reply:
Ah, I renamed `MaybeIbcAddress` to `TAddrData` and forgot to update this comment. Will fix, thanks.